### PR TITLE
State page updates for Katie

### PR DESCRIPTION
**Pull request by caleybrock**
These are mostly Katie's changes. I just did a build to make sure the haml didn't break staging locally (instead of her making large content changes in haml).

From an overall glance it looks good to me. Tested with FL, GA and AK.

### CODE REVIEW

**Comment by caleybrock** [comment]
FYI @Hamms @islemaster @ashercodeorg the error we saw on staging that happened > 3600 times was due to bad haml on staging (edited as content changes) that cause the pdf generation to fail it's attempts.

File: pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml [hunk 1]
```diff
@@ -192,76 +192,68 @@ theme: pdf
...
-                  teachers.
-      - if state_info[:num_ms_trained_i] + state_info[:num_hs_trained_i] != 0
-        %li
-          Code.org has provided professional learning for 
-          %b
-            = format_integer_with_commas(state_info[:num_ms_trained_i])
-            middle school
-            - if state_info[:num_ms_trained_i] == 1
+    %p
+      - if state_info[:professional_learning_partners_t] == 'none' && (state_info[:num_elementary_trained_i] != 0 || state_info[:num_ecs_csd_i] != 0 || state_info[:num_csp_i] != 0)
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml:204]: nit: maybe wrap with |s at line ends?

File: pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml [hunk 2]
```diff
@@ -192,76 +192,68 @@ theme: pdf
...
+        Code.org
+      - elsif state_info[:professional_learning_partners_t] != 'none'
+        Code.org, its regional partner(s)
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml:207]: nit: detect plural and only then show s?

File: pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml [hunk 3]
```diff
@@ -441,4 +433,3 @@ theme: pdf
       %p Launched in 2013, Code.org® is a non-profit dedicated to expanding access to computer science, and increasing participation by women and underrepresented students of color. Our vision is that every student in every school should have the opportunity to learn computer science.
     %p.clear-both.sources Data is from the Conference Board for job demand, the Bureau of Labor Statistics for state salary and national job projections data, the College Board for AP exam data, the National Center for Education Statistics for university graduate data, the Gallup and Google research study Education Trends in the State of Computer Science in U.S. K-12 Schools for schools that offer computer science and parent demand, and Code.org for its own courses, professional learning programs, and participation data.
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml:434]: nit: We could just wrap these ourselves and the browser should reconstruct the lines just fine.

**Comment by breville** [comment]
Interesting to ponder how to build some helpers or another system to make this more readable, but lgtm from scanning.

**Pull request merged**